api/python/src/cellxgene_ontology_guide/entities.py

```diff
@@ -28,3 +30,22 @@ class CuratedOntologyTermList(Enum):
     SYSTEM = "system"
     TISSUE_GENERAL = "tissue_general"
     UBERON_DEVELOPMENT_STAGE = "uberon_development_stage"
+
+
+class OntologyTreeNode:
+    """
+    Class to represent a node in an ontology term tree.
+    """
+
+    def __init__(
+        self,
+        term_id: str,
+        children: Optional[List["OntologyTreeNode"]] = None,
+        term_counter: Optional[Dict[str, int]] = None,
```

Comment:
This should be a `Counter`. If it can be a `dict`, then `self.term_counter` should be cast as `Counter` if it's not one already. 
```suggestion
        term_counter: Optional[Counter[str, int]] = None,
```